stdlib/universe/moving_average.go

```diff
@@ -2,10 +2,10 @@ package universe
 
 import (
 	"fmt"
+	"github.com/influxdata/flux/values"
```

Comment:
this should go with the second group of imports separated by space.

(You'll get used  to always check imports of  the files in  a commit before pushing :))